airflow/providers/google/marketing_platform/example_dags/example_display_video.py

```diff
@@ -99,9 +104,18 @@
 
     # [START howto_google_display_video_upload_line_items_operator]
     upload_line_items = GoogleDisplayVideo360UploadLineItemsOperator(
-        task_id="upload_line_items",
-        bucket_name=BUCKET,
-        object_name=OBJECT_NAME,
+        task_id="upload_line_items", bucket_name=BUCKET, object_name=OBJECT_NAME,
     )
-    # [END howto_google_display_video_upload_line_items_operator]
+    # [END howto_google_display_video_upload_multiple_entity_read_files_to_big_query]
+
+    # [START howto_google_display_video_upload_multiple_entity_read_files]
+    upload_erf_to_bq = GCSToBigQueryOperator(
+        task_id='upload_erf_to_bq',
+        bucket=BUCKET,
+        source_objects=[f"gdbm-{GMP_PARTNER_ID}/entity/{{{{ ds_nodash }}}}.*.{ENTITY_TYPE}.json"],
```

Comment:
Can we create a helper factory function that will help users easily generate those stings?
```python
def erf_bucket(partner_id, entity_type):
    return f"gdbm-{partner_id}/entity/{{{{ ds_nodash }}}}.*.{entity_type}.json"
```
WDYT?

Reply:
Sure!

Reply:
Once we 've got the method let's use it here :D